pfcpiface/messages.go

```diff
@@ -174,3 +196,32 @@ func (pConn *PFCPConn) sendPFCPRequestMessage(r *Request) (message.Message, bool
 		}
 	}
 }
+
+// dumpRawPFCP writes the raw PFCP packet bytes to a file under dumpDir.
+func dumpRawPFCP(dumpDir, addr string, buf []byte) error {
```

Comment:
Should we limit the amount of size the dump can use? Otherwise, disk/storage could be filled up without end-user knowing. What do you think?